### PR TITLE
Remove CodeReady from deactivation email template

### PR DESCRIPTION
**Pull request by alexeykazakov**
Also I've removed tests (and template preview generation) for templates we don't use anymore since there were failing (they relay on prod accounts which were deactivated).

I didn't remove the updated templates itself though since it would require deeper refactoring.

### CODE REVIEW

File: template/user.deactivation/body.html [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
-        margin: 30px 0;
-      }
-    </style>
-  </head>
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>
+    Notice: Your CodeReady Toolchain / OpenShift.io free account is inactive
```
Comment (xcoulon) [inline template/user.deactivation/body.html]:
```suggestion
Notice: Your OpenShift.io free account is inactive
```

File: preview/main.go [hunk 2]
```diff
@@ -38,20 +38,10 @@ func main() {
...
-		{"d28f8344-4956-497a-b43b-7f217087a931", string(types.CommentCreate)},
-		{"51d968b1-b9e5-4ec1-884a-ff256902c753", string(types.CommentCreate)},
-		{"51d968b1-b9e5-4ec1-884a-ff256902c753", string(types.CommentUpdate)},
-		{"3383826c-51e4-401b-9ccd-b898f7e2397d", string(types.UserEmailUpdate)},
-		{"f3a184bf-963b-448a-86b4-307ae455223b", string(types.UserDeactivation)},
-		{"81d1c3bf-fcf2-4c4e-9d12-f9e5c15fb9ab", string(types.InvitationTeamNoorg)},
-		{"297f2037-72e9-42b3-a5fc-76d843877163", string(types.InvitationSpaceNoorg)},
-		{"0a9c6814-462e-411c-8560-d74297bf1ceb", string(types.AnalyticsNotifyCVE)},
-		{"7dd737ed-c0fa-4f2f-9a3a-ed43778bb7fe", string(types.AnalyticsNotifyVersion)},
+		{"1a3496ca-edc0-42f7-958f-ba02ed3ef54d", string(types.UserEmailUpdate)},
```
Comment (xcoulon) [inline preview/main.go]: why do you use the same value everywhere?

File: template/template_test.go [hunk 3]
```diff
@@ -21,8 +21,8 @@ import (
 )
 
 const (
-	OpenshiftIOAPI     = "http://api.openshift.io"
-	OpenShiftIOAuthAPI = "https://auth.openshift.io"
+	OpenshiftIOAPI     = "http://api.prod-preview.openshift.io"
```
Comment (xcoulon) [inline template/template_test.go]: why do you need to change this?

File: preview/main.go (hunk 2 above)
Comment (alexeykazakov) [reply preview/main.go]: I’m using the same identity since it’s easier. Is it a problem?

File: template/template_test.go (hunk 3 above)
Comment (alexeykazakov) [reply template/template_test.go]: The same problem as in tests. Prod accounts may be deactivated (as they did for accounts used here before that PR).

Comment (xcoulon) [reply template/template_test.go]: ok

File: preview/main.go (hunk 2 above)
Comment (xcoulon) [reply preview/main.go]: not really. I was just curious.

**Pull request merged**